packages/integrations/src/builders/users/criteriaBuilder.ts

```diff
@@ -0,0 +1,59 @@
+import { UserQueryCriteria } from "@relewise/client";
+
+export class CriteriaBuilder {
+    
+    private userQueryCriteria: UserQueryCriteria;
+
+    constructor() {
+        this.userQueryCriteria = {};
+    }
+
+    public byAuthenticatedId(authenticatedId: string): this {
+        this.userQueryCriteria.authenticatedId = authenticatedId;
+
+        return this;
+    }
+
+    public beTemporaryId(temporaryId: string): this {
+        this.userQueryCriteria.temporaryId = temporaryId;
+
+        return this;
+    }
+
+    public byEmail(email: string): this {
+        this.userQueryCriteria.email = email;
+
+        return this;
+    }
+
+    public language(language: string): this {
+        this.userQueryCriteria.language = { value: language };
+
+        return this;
+    }
+
+    public currency(currency: string): this {
+        this.userQueryCriteria.currency = { value: currency };
+
+        return this;
+    }
```

Comment:
Place these at the end so all "byXXX" are "grouped" together, makes it easier to read what "by" options there are